src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +55,46 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/* Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = (element: Element | Document) => {
+  // Find the image elements to act on.
+  const timeElements = [].concat(
+    element instanceof HTMLTimeElement ? [element] : [],
+    Array.from(element.getElementsByTagName('time')),
+  );
+
+  timeElements.forEach(elem => {
+    const timeStamp = elem.getAttribute('datetime');
```

Comment:
Is there a reason not to use the more direct `elem.datetime` here?

If you're copying from the example of `rewriteImageUrls`, note that there's a comment there explaining the reason we sadly can't use `img.src` at the corresponding spot there.

Reply:
I was not aware that it's possible to refer to element-specific attributes directly.